frontend/src/contexts/reducer.js

```diff
@@ -159,6 +159,12 @@ const reducer = (state, action) => {
                 ...state,
                 selectedSuiteState: null,
             };
+        case 'flushParams':
```

Comment:
Just a comment, flushParams could be either used to remove all params, or named more accordingly if it's not mean't to flush all the Parameters there are.

Reply:
I can change this to flushQueryParams